src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/BackwardsCompatibility/Symfony6PrivateServicesPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\DependencyInjection\Compiler\BackwardsCompatibility;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/** @internal */
+final class Symfony6PrivateServicesPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        // Some services are still fetched directly from the container in both Sylius and its extracted bundles (e.g. ResourceBundle)
+        // Before we fix it let's make them public to keep backward compatibility
+
+        $this->makeServicePublic('form.factory', $container);
+        $this->makeServicePublic('security.authorization_checker', $container);
+        $this->makeServicePublic('security.authorization_checker', $container);
+        $this->makeServicePublic('security.csrf.token_manager', $container);
+        $this->makeServicePublic('security.csrf.token_manager', $container);
+        $this->makeServicePublic('security.token_storage', $container);
+        $this->makeServicePublic('security.token_storage', $container);
+        $this->makeServicePublic('validator', $container);
```

Comment:
Another hot-fix, but the easiest solution for now. We still use in some of our services a container injection :/ The problem is, some of these happen in Sylius itself, and some in extracted bundles (like Resource bundle, for example). So I make these services public temporarily, until we remove this abomination from the code (without breaking the backward compatibility, obviously 🎉)